pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -295,6 +313,10 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	// Any state needed by handler functions
 	ctx := ledManagerContext{}
 	ctx.countChange = make(chan types.LedBlinkCount)
+
+	if appStatusDisplayFunc != nil && model == "SIEMENS AG.SIMATIC IPC127E" {
```

Comment:
No need for this explicit model check

Reply:
The reason for that check is that in case in future if another model implements appStatusDisplayFunc and that has nothing to do with leds, so added an additional safety. If you think we should remove now and deal later, I am OK with that.